calico-cloud/_includes/components/EnvironmentFile.js

```diff
@@ -43,7 +43,7 @@ export default function EnvironmentFile(props) {
         </Admonition>
       )}
       <p>
-        Sample <code>EnvironmentFile</code> - save to <code>/kubernetes/calico/calico.env</code>
+        Sample <code>EnvironmentFile</code> - save to <code>/etc/calico/calico.env</code>
```

Comment:
Should this change also go in CC18?
Seems to go in a lot of places:
<img width="1028" alt="image" src="https://github.com/tigera/docs/assets/39195715/64eb8e89-db8b-4811-9df4-606a67c0282b">

Reply:
Nice catch! Pushed a commit.